src/main/java/duke/command/UserCommandManager.java

```diff
@@ -57,31 +59,31 @@ public static void handleUnknownCommand() {
      * @param userCommand type of action user wants to perform
      * @param userInput   arguments to the action (i.e. task description)
      */
-    
+
     public void handleCommands(String userCommand, String userInput) {
         Scanner scanner = new Scanner(userInput);
         switch (userCommand) {
-        case "list":
-            myList.printList();
-            break;
-        case "mark":
-            myList.markAsDone(scanner.nextInt() - 1);
-            break;
-        case "unmark":
-            myList.markAsUndone(scanner.nextInt() - 1);
-            break;
-        case "todo":
-            myList.addToList(scanner.nextLine());
-            break;
-        case "deadline":
-            handleDeadlineCommand(scanner.nextLine().trim().split("/by"));
-            break;
-        case "event":
-            handleEventCommand(userInput);
-            break;
-        default:
-            handleUnknownCommand();
-            break;
+            case "list":
```

Comment:
Seems like an issue with indentation after the changes as the cases for the switch statement are indented afterwards. Can you verify?

Reply:
Yup apparently i did not configure case indentation on my laptop ide